resources/js/src/app/components/myAccount/ChangePaymentMethod.js

```diff
@@ -36,10 +40,22 @@ Vue.component("change-payment-method", {
     {
         checkChangeAllowed()
         {
-            ApiService.get("/rest/io/order/payment", {orderId: this.currentOrder.order.id, paymentMethodId: this.paymentMethod})
+            ApiService.get("/rest/io/order/payment", {orderId: this.currentOrder.id, paymentMethodId: this.paymentMethod})
                 .done(response =>
                 {
-                    this.changePossible = response;
+                    // TODO: research - if response should be false, it returns an object
+                    if (response === true)
```

Comment:
this.changePossible = typeof response === "object" ? response.data : response;

damit kann man die 3 if else etc. anweisungen sparen